platform/platform_windows_386.go

```diff
@@ -41,3 +50,23 @@ func platGetVersion(outdata *byte) (maj uint64, min uint64, err error) {
 	min = uint64(info.wki100_ver_minor)
 	return
 }
+
+func platGetServerInfo(outdata []byte) (si101 SERVER_INFO_101){
+	var outdata []byte
+	outdata = (*[24]byte)(unsafe.Pointer(data))[:]
+	si101.sv101_platform_id = binary.LittleEndian.Uint32(outdata)
+
+	//stringptr := (*[]uint16)(unsafe.Pointer(uintptr(binary.LittleEndian.Uint64(outdata[4:]))))
```

Comment:
should we clean this up?

Reply:
Intentionally left (as in the existing code) to show why we just arbitrarily skpped over 4/8 bytes in the buffer.